### DISCUSSION TITLE
Welcome to Discussions! 👋

### DISCUSSION THREAD

**Discussion by Vdauphin** [Vdauphin/HeartsAndMinds#1005] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share tips.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.
  * Remember to be UpToDate check ✅ _Discussions_ in _Watch_ menu

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.
Use English only.

**Comment by GoldJohnKing** [comment]
It's so great we have a discussion thread now! 🎉

Hearts and Minds is the most popular mission ever in our community. It has deep integration with ACE framework. We love H&M especially for its logistics system (the vehicle wreck, the towing and lifting system and the logistics point). Its open world design is another part that attracted us, we feel as a real soldier from a real army fighting in a real battleground facing real threats.

As a server host in Chinese mainland, my major contribution is providing high quality Simplified Chinese translation for this awesome mission. I have introduced H&M to many Chinese communities, and thanks to @Vdauphin's help, we are able to make modifications to meet our specific needs. I also hold a forked repository on Github, which contains all the modified version for our fellow communities.

So if you are a player in China and have questions or issues or whatever, come to this discussion and at me (@GoldJohnKing) without any hesitate!

Thanks @Vdauphin again and again for making this awesome mission! 🥇

**Comment by JWB502** [comment]
HnM is the only game mode I play...for years now! I just enjoy having a dynamic mission without any intrusion (like a Zeus). Every now and then I'll play Zeus missions with people but most that have had the power will abuse it. So Hearts and Minds is the best option to create a dynamic battlefield. I mostly use the mission to create a Vietnam style mission or RHS, it just depends on the crowd.

**Reply by GoldJohnKing** [reply]
Yes! H&M is really easy to adapt or modify, because of its great coding style and file arrangement structure, as well as its detailed comments. And its framework covers almost every piece of aspects of Arma, and fits almost every kind of needs!

**Comment by MoMaeki00** [comment]
How can I force ace medical system on the server so that players can use the server settings

**Reply by Vdauphin** [reply]
You are off-topic, ask this here: https://slackin.ace3mod.com/

**Comment by mrtzkyle** [comment]
Hello, I would like to know how to add specific people as translators if possible. My mission is already started and a long way through.
Also while im here, how would I change the spawn on players and the logistics object spawn area with an already played map? thank you

**Reply by Vdauphin** [reply]
Hello,
You are off-topic, please use : https://github.com/Vdauphin/HeartsAndMinds/discussions/new
Cheers

**Comment by Todd1215** [comment]
Good afternoon and thank you for creating such a great MP mission. The ARMA group I play with all seem to really like the gameplay and side missions and really just the immersion factor that has been brought to this project. We are a small group and like to try and stick with real world tactics and logistics, it just makes for a better experience we feel. With that said we are running 1.21.1 and was wondering if applying version 1.21.3 will break the saved game from 1.21.1? If it will we will just finish up our current campaign and upgrade but it would be nice to be able to upgrade to eliminate the towing bugs and such.

thanks in advance!!!

**Reply by Vdauphin** [reply]
Hello,
Happy you like it!
No worries, data base isn't break between hotfix 

And even between major update because I have a file to convert old database to the new one. This last thing doesn't work between two big update like migrating from 1.19 to 1.21 but 1.19 to 1.20 then 1.20 to 1.21 will work

Cheers

**Comment by H4yn3x** [comment]
Hey, fellas. I can't figure out how to assign players as interpreters, since the role is not available (at least in mine). I'm on version 1.25.

**Reply by Rad-Ezz** [reply]
@H4yn3x 

Hello, in editing mode, there are three soldiers facing each other: a senior officer, an engineer, and an interpreter.
Simply double-click on the interpreter and you'll see an INIT code. You can copy and paste this code to assign it to other soldiers.

You can rename each soldier via their role in Attributes > Job Description.
This way, you'll see the roles in the lobby when players take their places.


See image

![20250510135728_1](https://github.com/user-attachments/assets/7712ca52-be3f-4c44-83b0-6890272f4b21)
